src/frontend/shortlist/src/App.vue

```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function markSchoolAsTrashed(payload) {
```

Comment:
I would suggest changing the console.log statements here as this function does not add schools to a list.